### PR TITLE
fix: function and params name word error in badge component

### PR DESCRIPTION
**Pull request by Rain120**
### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

Fix the functions and params name word error in badge component, here is an image about what I change.

image

### CODE REVIEW

**Review by abenhamdine — APPROVED** [review]

File: components/badge/index.tsx [hunk 1]
```diff
@@ -144,7 +144,7 @@ export default class Badge extends React.Component<BadgeProps, any> {
         data-show={!hidden}
         className={scrollNumberCls}
         count={displayCount}
-        displayComponent={this.renderDispayComponent()} // <Badge status="success" count={<Icon type="xxx" />}></Badge>
+        displayComponent={this.getNumberedDisplayCount()} // <Badge status="success" count={<Icon type="xxx" />}></Badge>
```
Comment (abenhamdine) [inline components/badge/index.tsx]: why that change ?
renderDispayComponent !== getNumberedDispayCount

**Comment by yoyo837** [comment]
Please update the snapshot.

File: components/badge/index.tsx (hunk 1 above)
Comment (Rain120) [reply components/badge/index.tsx]: It's my fault, I would fix it and recommit it

**Comment by yoyo837** [comment]
CI fails

**Comment by Rain120** [comment]
@yoyo837 I'll check it.

**Comment by yoyo837** [comment]
@afc163  Should this be moved to 4.0？

**Comment by afc163** [comment]
We can fix it in master.

**Comment by afc163** [comment]
Fixed in master.

**Pull request closed without merging**